src/main/resources/db/migration/V10__add_plan_auditing.sql

```diff
@@ -0,0 +1,14 @@
+create table if not exists practitioner
+(
+    id       serial           PRIMARY KEY,
+    uuid     varchar(255)     NOT NULL UNIQUE,
```

Comment:
We use the `uuid` type for uuid columns everywhere else, so why have this as a `varchar`? And why so long if it's a `uuid`?

Reply:
Yep, it's not really a UUID from what I've seen of test data moving back and forth between Capita and our current deployments (on SAN this is). It's usually a identifier that seems like it might be partially based on the users name.

Reply:
Renamed in https://github.com/ministryofjustice/hmpps-sentence-plan/pull/81/commits/0e13abe81928a4e6e31ccfa42b052d6bfa0b8259